RevenueCatUI/Views/DebugErrorView.swift

```diff
@@ -78,7 +78,6 @@ struct DebugErrorView<Content: View>: View {
         Text(self.description)
             .unredacted()
             .padding()
-            .fixedSize(horizontal: true, vertical: false)
```

Comment:
I _think_ I've used this before for making sure that there are no weird cutoff overflow things with the text 🤔 was this actually not needed?

Reply:
When this was here a long error message was constrained to a single line, so it would get cut off at the beginning and end as the whole thing remained centred. Removing this makes it wrap to multiple lines.